docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -367,12 +367,24 @@ statements, the connector supports the following features:
 - {doc}`/sql/create-table`, see also {ref}`delta-lake-sql-basic-usage`
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
-- {doc}`/sql/alter-table`
+- {doc}`/sql/alter-table`, see also {ref}`delta-lake-alter-table`
 - {doc}`/sql/drop-schema`
 - {doc}`/sql/show-create-schema`
 - {doc}`/sql/show-create-table`
 - {doc}`/sql/comment`
 
+
+(delta-lake-alter-table)=
+
+### ALTER TABLE statement limitations
+
+The connector only supports the `ALTER TABLE RENAME TO` statement when met with
+the following conditions:
+
+* The table type is external.
```

Comment:
can we link to any section that explains what an external table is?

Reply:
There's a section in the [Accumulo connector](https://trino.io/docs/current/connector/accumulo.html#external-tables) about external tables. Can I make it more generic, convert it into a fragment, and then place it above the **Table properties** in both connector docs?

Reply:
No.. the accumulo stuff only applies on the high level.. maybe explain in the register table section of the Delta Lake connector (https://trino.io/docs/current/connector/delta-lake.html#register-table) or just above there where it talks about external systems

Reply:
If other connectors support creating external tables, why is it only mentioned in connection with Accumulo? Documentation on external tables should be a standalone document or section we can link to.